shared/app/tab-bar/index.render.desktop.js

```diff
@@ -73,7 +73,7 @@ const TabBarRender = ({onTabClick, selectedTab, username, badgeNumbers}: Props)
 
 const stylesTabBar = {
   ...globalStyles.flexBoxColumn,
-  backgroundColor: globalColors.darkBlue2,
+  backgroundColor: 'red', // TEMP while testing globalColors.darkBlue2,
```

Comment:
If this is the last one, can we remove this?

Reply:
i'll remove it when we merge it down